### DISCUSSION TITLE
Simple version for iOS - even if just to merge videos

### DISCUSSION THREAD

**Discussion by darinhouston** [mifi/lossless-cut#1802] [Ideas]
I film a lot of sports events for game film and then share with team and coach and families and so forth - I start and stop during periods of inactivity to shorten the videos, save space, and make them more watchable.  That leads to 20-30 or more short snippets of video - my workflow is to wait until I get home, import all that video to the computer, load it into lossless-cut, then merge and save, then share.  For highlights, I then load again and do some in/out marking as well before saving again.  But, that's not critical.

There simply isn't any existing lossless merging tools that aren't larger video editing platforms (like iMovie) and which don't require importing and exporting and conversion and so forth.

How hard would it be to create a version with minimal UI to wrap the tools/libraries to just select a number of videos from photo library and save a combined version?

The built-in Photos app does a great job on splitting and cropping video, but doesn't combine them.

**Comment by mifi** [comment]
I added your request here: #818
it’s indeed possible but I don’t have time these days.
